### PR TITLE
Feature/new design request strains

### PR DESCRIPTION
**Pull request by orzut**
Please test a cart for all the following user stories (and for any edge cases you come up with):
As a [ ROLE ] I want to do [ACTION]. 
- As a non-logged user, I want to initiate a cart and be able to add/remove stuff to it.
[PROMOTION] of a cart to be assigned to a user. 
- As a non-logged user, I want to initiate a cart and be able to add/remove stuff to it, and to login to the site and have the cart be updated with my user id/email. This way the cart knows it belongs to me. 
- As a logged-in user, when i navigate to the order page, I want my existing cart to show up (if it exists). 
- If a user logs in on a second device, I want my existing cart to show up. 
- As a non-logged-in user in a second device, [@vince] 
- As a non-logged-in, I want to navigate to the order page. A new cart is created and it is empty. I then login, the cookie will map to an empty cart (unassigned to a user), and my previously stored cart is now populated. [edge case].

### CODE REVIEW

**Review by r-vieira — APPROVED** [review]
looks good!

File: src/modules/site-v2/templates/_includes/footer.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <div class="container-fluid mt-5 p-5">
     <div class="d-flex flex-sm-column flex-md-row flex-wrap justify-content-center pt-5">
       <div class="col-12 col-md-3 mb-5">  
-        <a href="/"><img src="/static/img/caeNDRLogo.png" class="d-block img-fluid mx-auto footerImage"></a>
+        <a href="/"><img src="/static/img/caeNDRLogo.png" class="d-block img-fluid mx-auto footerImage" alt="Caendr Logo"></a>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/_includes/footer.html]: Thanks for adding the alt text here! I think that "Home" would make more sense here, because the logo allows the user to go back to the homepage, so we'd want a way to communicate that to users of screen readers. Otherwise, the logo doesn't communicate any information to a screen-reader user ([more info](https://www.smashingmagazine.com/2021/06/img-alt-attribute-alternate-description-decorative/#supplemental-icons/))

File: src/modules/site-v2/templates/_includes/navbar.html [hunk 2]
```diff
@@ -2,23 +2,24 @@
...
+
 </style>
 
 <header>
   <!-- Global Navbar -->
   <nav class="navbar navbar-expand-lg fw-bold mt-3">
     <div class="container-fluid">
       <div class="col-3 col-md-1 navbar-brand ms-5">
-        <a href="/"><img src="/static/img/caeNDRLogo.png" class="img-fluid"></a>
+        <a href="/"><img src="/static/img/caeNDRLogo.png" class="img-fluid" alt="Caendr Logo"></a>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/_includes/navbar.html]: Same comment here -- I think it makes more sense to use "Home" as the alt text.

File: src/modules/site-v2/templates/order/order.html [hunk 3]
```diff
@@ -1,122 +1,323 @@
...
+        margin: 1rem;
+
+    }
+</style>
+{% endblock %}
 
-        <table class="table table-hover table-striped" style="table-layout: fixed">
+{% block content %}
+    {% if len(cartItems) %}
+    <div id="tableContainer">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html]: In general, it's best practice to use Bootstrap for layouts and styling and to avoid creating custom CSS if you don't have to. It can be really helpful to refer to the design system to see how similar styles were achieved there. For this section, I'd recommend taking a look at the [Request Strains pattern](https://github.com/northwestern-mti/caendrbootstrap/blob/main/src/requestStrains.html) to see how the grey background boxes were achieved. 

So here, instead of using the `tableContainer` custom CSS, I'd recommend using the following: 

```
<div class="col-12 col-md-9 p-5 mb-5 mx-auto  rounded shadow-sm text-bg-light">
...
</div>
```

- `col-12` and `col-md-9` use the [Bootstrap grid system](https://getbootstrap.com/docs/5.2/layout/grid/#grid-options) to control the width of the container. `col-12` tells the container to take up the full width of the grid (12 columns) on extra small devices (mobile). `col-md-9` tells the container to take up 9 columns of width on medium and larger devices (desktop).
- `p-5` sets the padding at the highest level (1 is lowest, 5 is highest) on all sides of the container. 
- `mb-5` sets margin-bottom on the container
- `mx-auto` will automatically horizontally center the container
- `rounded` creates the slightly rounded corners
- `shadow-sm` adds a small amount of drop shadow
- `text-bg-light` sets the background color

Using these classes will make it easier to change things on a global level if we need to at a later date. For example, if we decide to change the color of `text-bg-light` in a future redesign, we can do that by simply editing the Bootstrap CSS files, instead of having to change hex values on several different pages.

File: src/modules/site-v2/templates/order/order.html [hunk 4]
```diff
@@ -1,122 +1,323 @@
...
+        <table class="table table-striped m-4" id="orderTable">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html]: Please be sure to wrap all tables in a `table-responsive` div to ensure tables adjust appropriately across different screen sizes. Plus, you then won't need to set the max-width with custom CSS. 

We should also include a caption for accessibility, to describe the purpose of the table for users of screen readers.

```
<div class="table-responsive">
    <caption class="visually-hidden">A list of items included in your shopping cart.</caption>
    <table> ... </table>
</div>
```
Might also be good to remove `m-4`, as I noticed it causes the table to "wiggle" once you make the table responsive.

File: src/modules/site-v2/templates/order/order.html [hunk 5]
```diff
@@ -1,122 +1,323 @@
...
+        max-width: 800px;
+    }
+    #orderTable {
+        max-width: 650px;
+    }
+    #orderTable th {
+        border-bottom: 1px solid black
+    }
+    #info {
+        background-color: #d9dbf8;
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html]: You can use Bootstrap variables to ensure that colors remain consistent when there's not an easy Bootstrap class available (I know there isn't for this color). This is the same color as the footer, so you could use `background-color:var(--bs-footerBg)`.

File: src/modules/site-v2/templates/order/order.html [hunk 6]
```diff
@@ -1,122 +1,323 @@
...
+        color: white;
+        width: 80%;
+        margin-left: 50px;
+    }
+    #shippingInfo, #paymentInfo {
+        background-color: #e9e9e9;
+        padding: 40px;
+        margin: 30px
+    }
+    h3 {
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html]: I don't think you need this, because I don't see any `<h3>`s on the page. Plus, it's conflicting with the text in the footer, causing the `<h3>`s there (tools, data, about us, cite CaeNDR) to be oddly centered.

File: src/modules/site-v2/templates/order/order.html [hunk 7]
```diff
@@ -1,122 +1,323 @@
...
-            {% endfor %}
 
-            {{ form.csrf_token }}
-            {{ render_field(form.name) }}
-            {{ render_field(form.phone, placeholder="(847)-867-5309") }}
-            {{ render_field(form.address, placeholder="123 Sheridan Rd.\nChicago IL. 60657") }}
-            {{ render_field(form.email) }}
+<form id='form-submit' method="POST">
+    <div class="row">
+        <div class="col">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html]: Again, I'd use Bootstrap on this div to add the styling, instead of using custom css on the `shippingInfo` div below. 

This should also mean you won't need to use the custom CSS with `form-submit` to set the margin.

```
 <div class="col text-bg-light shadow-sm rounded p-5 m-3">
...
</div>
```

File: src/modules/site-v2/templates/order/order.html [hunk 8]
```diff
@@ -1,122 +1,323 @@
...
+            </div>
         </div>
-
-        <div class="col-lg-6">
-            {{ render_field(form.shipping_service) }}
-            {{ render_field(form.shipping_account) }}
-            {{ render_field(form.payment) }}
-            {{ render_field(form.comments) }}
-            <small>If you choose to pay by credit card, an account will be generated for you in Northwestern's Core Facility, NUcore. If there is a business administration person who will oversee the payment, please include that person's name and email address in the comments field. They will be added to your NUcore account.</small>
+        <div class="col">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html]: Same comment here as the "Shipping Info" div. I'd recommend the below instead of custom CSS: 

```
 <div class="col text-bg-light shadow-sm rounded p-5 m-3">
...
</div>
```

File: src/modules/site-v2/templates/order/order.html [hunk 9]
```diff
@@ -1,122 +1,323 @@
...
+            <div>
+                <h2>Payment Information</h2>
+                <div id="paymentInfo">
+                    <div class="form-group">
+                        {{ form.payment.label}}
+                        <button type="button" class="icon-btn" data-toggle="modal" data-target="#paymentInfoModal" aria-label="Payment information"><i class="bi bi-info-circle-fill" id="payment-info-icon" aria-hidden="true"></i></button>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html]: You don't need custom CSS to set the color of the icon. Remember that an icon here is just text, so you can use all the usual Bootstrap classes to set text color. This will help ensure the colors used are consistent across the site.

`<i class="bi bi-info-circle-fill text-secondary" aria-hidden="true"></i>`

File: src/modules/site-v2/templates/order/order.html [hunk 10]
```diff
@@ -1,122 +1,323 @@
...
-                <tr>
+                {% for item in cartItems %}
+                <tr class='cartItem'>
                     <td>{{ loop.index }}</td>
-                    <td>{{ item }}</td>
-                    <td class="price">${{'%0.2f'| format(price|float)}}</td>
+                    <td><em>{{ item.species }}</em></td>
+                    <td>{{ item.name }}</td>
+                    <td>${{ item.price }}</td>
+                    <td class="delete">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html:63]: I don't think you need custom css here. All buttons should use the `btn` class for Bootstrap, which will remove the weird grey box. And then you can use `text-danger` to set the icon color. 

```
<button type="button" class="btn" info={{ item.name }}>
   <i class="bi bi-trash-fill text-danger" class="delete-icon" aria-hidden="true"></I>Delete
</button>
```

File: src/modules/site-v2/templates/order/order.html [hunk 11]
```diff
@@ -1,122 +1,323 @@
...
-                <button value="Checkout" class="btn btn-nu btn-block" id="checkout" ><span class="glyphicon glyphicon-shopping-cart" aria-hidden="true"></span> Order</button>
-                <br />
-            </div>{# /col-sm-4 #}
-            </div>{# /form-group #}
+<div class="modal fade" id="paymentInfoModal" tabindex="-1" role="dialog" aria-labelledby="paymentInfoTitle" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="paymentInfoTitle">Payment Information</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html:133]: Please be sure to use the `btn-close` class to ensure this is styled consistently with other modals used on other pages.

File: src/modules/site-v2/templates/order/order.html [hunk 12]
```diff
@@ -1,122 +1,323 @@
...
-                Industry Order <span class="glyphicon glyphicon-new-window" aria-hidden="true"></span>
-            </button>
+    </div>
+</div>
+<div class="modal fade" id="deleteModal" tabindex="-1" role="dialog" aria-labelledby="deleteModalTitle" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="deleteModalTitle">Are you sure you want to delete this item?</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html]: Please be sure to use the `btn-close` class to ensure this is styled consistently with other modals used on other pages.

File: src/modules/site-v2/templates/order/order.html [hunk 13]
```diff
@@ -1,122 +1,323 @@
...
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <p>Are you sure you want to remove this item from your cart?</p>
+                <div class="d-flex justify-content-around p-2">
+                    <button type="button" class="btn btn-info" id="goBack">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order.html:153]: I'd recommend not using custom CSS for the text color here. Usually, if you want to set the text color on a button, I'd recommend doing the following: 

```
<button type="button" class="btn btn-info text-light" id="goBack">
     <i class="bi bi-slash-circle-fill"></I> No, go back.
</button>
```

But looking at this now, I think it's best to leave the text black (what happens when you remove the custom CSS) for accessibility purposes.

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 14]
```diff
@@ -1,144 +1,112 @@
...
-
-  <form>
-    <input type="button" class='btn btn-primary btn-block hidden-print' value="Print Invoice" onClick="window.print()">
-  </form>
-    
-</div>
-
-
 
 <div class='col lg-9 col-md-9 col-sm-9 col-xs-9'>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: I think there's too many responsive column sizes declared here that aren't needed. All that should be needed is: 

```
<div class="col col-md-9 mx-auto">
...
</div>
```
The `mx-auto` will also make sure that this is centered correctly on the page.

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 15]
```diff
@@ -1,144 +1,112 @@
...
 
   <div class="panel panel-default">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: `panel` isn't supported in Bootstrap 5 anymore, so I don't think this is adding anything stylistically, so this `div` can be removed.

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 16]
```diff
@@ -1,144 +1,112 @@
...
-      <div class="panel-heading">
-          <h3 class="panel-title text-center">{{ title }}</h3>
+      <div class="d-flex justify-content-between mb-4 panel-heading">
+        <h2 class="panel-title">{{ invoice }}</h2>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: You can remove `panel-title`.

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 17]
```diff
@@ -1,144 +1,112 @@
...
-</div>
-
-
 
 <div class='col lg-9 col-md-9 col-sm-9 col-xs-9'>
 
   <div class="panel panel-default">
-      <div class="panel-heading">
-          <h3 class="panel-title text-center">{{ title }}</h3>
+      <div class="d-flex justify-content-between mb-4 panel-heading">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: You can remove `panel-heading`

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 18]
```diff
@@ -1,144 +1,112 @@
...
+        <h2 class="panel-title">{{ invoice }}</h2>
+        <form>
+          <button type="button" class='btn btn-primary btn-block hidden-print' onClick="window.print()">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: Looks like `hidden-print` is no longer supported in Bootstrap 5. Should use `d-print-none` instead ([more info](https://getbootstrap.com/docs/5.2/utilities/display/#display-in-print).)

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 19]
```diff
@@ -1,144 +1,112 @@
...
-  </div>{# /panel #}
-
-
-
-
-<div class="row">
-    <div class="col-md-12">
-   <table class="table table-hover table-striped" style="table-layout: fixed">
+      {% if order_obj['is_donation'] == 'FALSE' %}
+      <div id="container">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: Same as similar comments made elsewhere, I'd avoid using custom CSS if possible. Should use the Bootstrap classes listed below instead: 

```
<div class="text-bg-light rounded shadow-sm p-5 mb-5">
     ...
</div>
```

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 20]
```diff
@@ -1,144 +1,112 @@
...
+          <p><strong>Order Date: {{ order_obj['date'] }}</strong></p>
+        </div>
+        
+        <p>{{ order_obj['name'] }}<br />{{ order_obj['address'].replace("\n", "<br />")|safe }}</p>
+        <br />
+        <p><strong>Shipping Service: </strong>{{ order_obj["shipping_service"] }}</p>
+        <br/>
+        <div>
+          <h3>Order Information</h3>
+          <table class="table table-hover table-striped" style="table-layout: fixed">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: Please be sure to wrap all tables in a responsive div and include a caption for accessibility.

In addition, if you make the table responsive and just add some Bootstrap padding, you won't need the custom CSS to control the width and margin for the table.

```
<div class="table-responsive px-5">
          <table class="table table-hover table-striped">
             <caption class="visually-hidden">A list of items ordered</caption>
              ...
          </table>
</div>
```

- `px-5` allows you to set the padding on only the left (start) and right (end) of the container.

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 21]
```diff
@@ -1,144 +1,112 @@
...
-      </div>{# /col-md-12 #}
-  </div>{# /row #}
+          </table>
+        </div>
+      </div>
+      {% endif %}
+    </div>{# /panel #}
+
+    <div class="m-4">
+      <h4>To complete your order</h4>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: This should be an `<h2>` for accessibility.

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 22]
```diff
@@ -1,144 +1,112 @@
...
+
+      {% else %}
+
+      <ol>
+          <li>Print this page out and please include it with your check.</li>
+          <li>Make a check payable to Northwestern University and write 'CeNDR' and your invoice number in the memo field.</li>
+          <li>Mail the check and this printed page to the address below.</li>
+      </ol>
+      <br />
+      <div id="mailInfo">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: Similar to other comments, you can use the following Bootstrap to style everything here (except for the background color - will comment on that above shortly). 

```
<div class="ps-5">
      <div class="border-start border-primary border-2 p-3">
           ...
      </div>
</div>
```

- I added the outer `div` with the class of `ps-5` to achieve the indent effect. This sets the padding for the left (start) side of the container
- `border-start` adds border only on the left (start) side of the container
- `border-primary` sets the color
- `border-2` controls the border width

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 23]
```diff
@@ -1,144 +1,112 @@
 {% extends "_layouts/default.html" %}
 
+{% block custom_head %}
+<style>
+    #container {
+      background-color: #e9e9e9;
+      padding: 20px;
+    }
+    #mailInfo {
+      background-color: #d9dbf8;
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: I'd use the Bootstrap variable to ensure that the color is consistent across the site. 

`background-color: var(--bs-footerBg);`

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 24]
```diff
@@ -1,144 +1,112 @@
...
+      <div id="mailInfo">
+        <p class='visible-print'>Mail a check payable to Northwestern University and a copy of your invoice to the address below. Please include your invoice number on the check.</p>
+        <address>
+        <strong>Erik C. Andersen, Ph.D.</strong><br />
+        c/o Life Sciences Financial Services<br />
+        Hogan Hall<br />
+        2205 Tech Dr, Suite 2-150<br />
+        Evanston IL, 60208
+        </address>
+        <p class='hidden-print'>Please contact us if you have any questions (<a href='mailto:cendr@northwestern.edu'>cendr@northwestern.edu</a>; 847-467-4382).</p>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: Similar to other comments, need to use `d-print-none` now.

File: src/modules/site-v2/templates/order/order_confirm.html [hunk 25]
```diff
@@ -1,144 +1,112 @@
...
+      <h4>To complete your order</h4>
+  
+      {% if order_obj['payment'] == 'credit_card' %}
+      <p>We will follow-up with you directly via email/phone to complete your order.</p>
+
+      {% else %}
+
+      <ol>
+          <li>Print this page out and please include it with your check.</li>
+          <li>Make a check payable to Northwestern University and write 'CeNDR' and your invoice number in the memo field.</li>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/order/order_confirm.html]: Should be "CaeNDR"

File: src/modules/site-v2/templates/strain/landing_page.html [hunk 26]
```diff
@@ -2,12 +2,60 @@
...
+        .card {
+            width:auto
+        }
+    }
+</style>
 
-<div>
-    <h2>Request Strains Landing Page</h2>
+<div class="container-fluid">
+    <h1>Request Strains</h1>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/strain/landing_page.html]: It's probably best to set the title of the page programmatically in strains.py to be consistent with the rest of the site and to make proper use of the default.html layout rather than declaring it explicitly here.

File: src/modules/site-v2/templates/strain/landing_page.html [hunk 27]
```diff
@@ -2,12 +2,60 @@
 
 {% block content %}
 
+<style>
+    h2, h3 {
```
Comment (natalieroman) [inline src/modules/site-v2/templates/strain/landing_page.html]: This custom CSS is creating some conflicts with the footer text so should be removed. I'll add some comments below on how to achieve the same effects using Bootstrap classes.

File: src/modules/site-v2/templates/strain/landing_page.html [hunk 28]
```diff
@@ -2,12 +2,60 @@
...
+            width:auto
+        }
+    }
+</style>
 
-<div>
-    <h2>Request Strains Landing Page</h2>
+<div class="container-fluid">
+    <h1>Request Strains</h1>
+    <h2>Please choose your affiliation:</h2>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/strain/landing_page.html]: Should use the following instead of custom CSS: 

`<h2 class="text-dark text-center"> ... </h2>`

File: src/modules/site-v2/templates/strain/landing_page.html [hunk 29]
```diff
@@ -2,12 +2,60 @@
...
+    <div class="container">
+        <div class="card shadow cardBorderBottom">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/strain/landing_page.html]: Would recommend adding some padding to make these cards look a little bit more consistent with cards used elsewhere on the site. 

```
<div class="card py-5 shadow cardBorderBottom">
     ...
</div>
```
- `py-5` sets padding only on the top and bottom of the container

File: src/modules/site-v2/templates/strain/landing_page.html [hunk 30]
```diff
@@ -2,12 +2,60 @@
...
+            <a href="{{ url_for('request_strains.strains_catalog')}}">
+                <i class="bi bi-book-fill text-info" aria-hidden="true"></i>
+                <div class="card-body">
+                    <h3 class="card-title">Academia</h3>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/strain/landing_page.html]: Would recommend using the below instead of custom CSS: 

`<h3 class="card-title text-dark"> ... </h3>`

File: src/modules/site-v2/templates/strain/landing_page.html [hunk 31]
```diff
@@ -2,12 +2,60 @@
...
+                    <p class="card-text">I am a researcher, post-doc, graduate student, undergraduate student, staff member, or other affiliate of a post-secondary educational institution.</p>
+                </div>
+            </a>
+        </div>
+        <div class="card shadow cardBorderBottom">
+            <a href="https://nulive.technologypublisher.com/tech/The_Caenorhabditis_Natural_Diversity_Resource">
+                <i class="bi bi-briefcase-fill text-info" aria-hidden="true"></i>
+                <div class="card-body">
+                    <h3 class="card-title">Industry</h3>
```
Comment (natalieroman) [inline src/modules/site-v2/templates/strain/landing_page.html]: Would recommend using the below instead of custom CSS: 

`<h3 class="card-title text-dark"> ... </h3>`

File: src/modules/site-v2/templates/strain/landing_page.html [hunk 32]
```diff
@@ -2,12 +2,60 @@
...
+        <div class="card shadow cardBorderBottom">
+            <a href="{{ url_for('request_strains.strains_catalog')}}">
+                <i class="bi bi-book-fill text-info" aria-hidden="true"></i>
+                <div class="card-body">
+                    <h3 class="card-title">Academia</h3>
+                    <p class="card-text">I am a researcher, post-doc, graduate student, undergraduate student, staff member, or other affiliate of a post-secondary educational institution.</p>
+                </div>
+            </a>
+        </div>
+        <div class="card shadow cardBorderBottom">
```
Comment (natalieroman) [inline src/modules/site-v2/templates/strain/landing_page.html]: Would recommend adding some padding to make these cards look a little bit more consistent with cards used elsewhere on the site. 

```
<div class="card py-5 shadow cardBorderBottom">
     ...
</div>
```
- `py-5` sets padding only on the top and bottom of the container

File: src/modules/site-v2/templates/strain/catalog.html
Comment (natalieroman) [inline src/modules/site-v2/templates/strain/catalog.html:1]: This looks great! However I noticed that there are some responsive layout issues, especially on the "Individual Strains" tab (see screenshot below). 

I addressed these issues in the ["Request Strains" page pattern](https://github.com/northwestern-mti/caendrbootstrap/blob/main/src/requestStrains.html) in the design system. That page pattern also reduces the need for writing custom CSS. 

Rather than trying to address things on a line-by-line basis, let's review the design system for this page together so I can give some responsive layout tips that'll be easier for me to explain face-to-face (and possibly with the aid of a whiteboard because I like visuals). 🙂

image

**Review by natalieroman — CHANGES_REQUESTED** [review]
Hi Orzu, this looks great! 

Sorry for leaving so many comments. I realized when I was reviewing this that when I handed off the designs to you, I didn't do a great job of including all the Bootstrap info you'd need to do this without custom CSS. And I maybe didn't do a great job of explaining how to use the Bootstrap grid for responsive layouts. So apologies for that-- I'm still learning a lot here. 🙂

I suggested in a comment that we discuss the Strain Catalog page face-to-face to go over some tips for responsive design, so maybe we can chat in-person on Wednesday. As always, please let me know if any of my comments are confusing or unclear!

**Review by vlagrassa — APPROVED** [review]
Looks good to me!

**Review by natalieroman — APPROVED** [review]
Looks great! Thanks for making all the markup/styling changes.

**Review by r-vieira — APPROVED** [review]
Looks good!
